### ISSUE TITLE
Optimize Screen Space

### ISSUE THREAD

**Issue by pawelsalawa** [pawelsalawa/sqlitestudio#2706]
_(This issue was migrated from the old bug tracker of SQLiteStudio)_

Original ID from old bug tracker: 2819
Originally created at: Tue Mar 31 10:50:11 2015
Originally last updated at: Tue Mar 31 10:50:11 2015

Hi and thanks for SQLiteStudio :)

Can you please do something to make better use of the available screen real-estate?

i.e. 
 - less chunky borders around everything
 - there are about 185 pixels used before the text for the first table name is displayed in the treeview!
- 200 vertical pixels used before the first row of my query results are visible!

Basically, the way I see it is that the app is about working with data - I want to see as much of that as possible, please. As is, on a 1280x720 screen (which I;m sometimes forced to work with) the app is difficult to work with, unfortunately.

Thanks for considering and keep up the great work :)\n\n**Operating system:**\nWindows XP/2003/2000

**Comment by pawelsalawa** [comment]
Originally created at: Tue Mar 31 10:50:11 2015

I agree. Unfortunately Qt and it's pretty themes forced this for start. I will need to tune this.

**Comment by pawelsalawa** [comment]
Originally created at: Tue Mar 31 10:50:11 2015

_Status changed to **assigned**_

**Comment by pawelsalawa** [comment]
Originally created at: Tue Mar 31 10:50:11 2015

_Status changed to **done**_
